esp/src/src-react/components/WorkunitSummary.tsx

```diff
@@ -110,6 +116,35 @@ export const WorkunitSummary: React.FunctionComponent<WorkunitSummaryProps> = ({
         setOtSpanId(otTrace.spanId);
     }, [otTraceParent]);
 
+    // Load stored issue links for this WUID (localKeyValStore)
+    React.useEffect(() => {
+        const store = localKeyValStore();
+        const key = `${ISSUE_LINKS_PREFIX}${wuid}`;
+        if (store) {
```

Comment:
Where was the store being written to?  I am not sure there is any value in keeping a local list of issues for a given wu adds any value?